### PR TITLE
UX-1189, UX-1214/tests for App section

### PR DESCRIPTION
**Pull request by Prathameshs**
deploy-infra-repository
)

### CODE REVIEW

**Comment by Prathameshs** [comment]
edit_deployed_app

File: cypress/integration/apps/edit-deployed-app.spec.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+     cy.intercept('**/releases').as('getReleases')
+     cy.intercept('**/releases/*').as('getRelease')
+     cy.intercept('**/clusters').as('getClusters')
+     cy.intercept('**/pods').as('getPods')
+     cy.intercept('**/namespaces').as('getNamespaces')
+   })
+
+   it('should be able to navigate deployed apps', () => {
+
+     const apiCalls = ['getCharts','1', 'getReleases']
```
Comment (jruddell) [inline cypress/integration/apps/edit-deployed-app.spec.ts:28]: what is the purpose of the `1` here?

Comment (Prathameshs) [reply cypress/integration/apps/edit-deployed-app.spec.ts:28]: We have dividing the API calls in 2 sections, main section and sub-section. so we are using the 1 to divide the API calls between 2 sections.

Comment (jruddell) [reply cypress/integration/apps/edit-deployed-app.spec.ts:28]: I am not a fan of this pattern. Why do you need to divide them? and how does a `1` accomplish this divide? The reason why I dont like it is it requires some form of implicit knowledge of how this should work verses some explicit action that is very clear what it is doing.

Comment (Prathameshs) [reply cypress/integration/apps/edit-deployed-app.spec.ts]: Resolved.

**Review by jruddell — APPROVED** [review]

**Pull request merged**